example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -177,7 +177,21 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
     });
   });
 
-  xdescribe('the PostToCmr task', () => {
+  describe('the MoveGranules task', () => {
+    let MoveGranulesTaskOutput;
+
+    beforeAll(async () => {
+      MoveGranulesTaskOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'MoveGranules');
+    });
+
+    it('has moved the files', () => {
+      const movedFiles = MoveGranulesTaskOutput.payload.granules[0].files.map((file) => file.filename);
+      expect(movedFiles).not.toContain(s3ummJsonFileLocation);
+      s3ummJsonFileLocation = movedFiles.filter((file) => file.includes('.cmr.json'))[0];
```

Comment:
the ummJson file is moved so we need to get the updated location later when we test the API.